### PR TITLE
Feat: APP-1694 Minimum Proposal Threshold settings.

### PR DESCRIPTION
**Pull request by WissenIstNacht**
## Description

Allow to change the minimum proposal threshold parameter as part of the normal settings flow for MV DAOs.

Task: [1694](https://aragonassociation.atlassian.net/browse/APP-1694)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: packages/web-app/src/pages/editSettings.tsx [hunk 1]
```diff
@@ -322,6 +361,20 @@ const EditSettings: React.FC = () => {
                 </AccordionContent>
               </AccordionItem>
 
+              <AccordionItem
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/editSettings.tsx:364]: It's slightly confusing having this as a community change when it appears under governance settings in the actions & action cards, but I'm guessing that's a design thing.

**Review by Fabricevladimir — APPROVED** [review]
Looks good 👍🏾

File: packages/web-app/src/pages/editSettings.tsx (hunk 1 above)
Comment (WissenIstNacht) [reply packages/web-app/src/pages/editSettings.tsx:364]: Horrible. But got it confirmed by Selim. The idea being that it needs to align with the DAO creation and the settings views.

**Review by jamesej — CHANGES_REQUESTED** [review]
I came across a minor display issue in that the New Settings display has a decimal '10.0' and the Old Settings display has none
ProposeSettingsOldSettingsProposalThreshold
ProposeSettingsNewSettingsProposalThreshold
 '10'. It might be that the actual new value has a decimal rounding issue and is very slightly larger than 10.

**Review by jamesej — APPROVED** [review]
Thanks Vali all good now

**Pull request merged**